forc/src/cli/plugin.rs

```diff
@@ -14,14 +15,29 @@ use std::{
 ///
 /// E.g. given `foo bar baz` where `foo` is an unrecognized subcommand to `forc`, tries to execute
 /// `forc-foo bar baz`.
-pub(crate) fn execute_external_subcommand(args: Vec<String>) -> Result<process::Output> {
+pub(crate) fn execute_external_subcommand(
+    args: Vec<String>,
+    silent: bool,
+) -> Result<process::Output> {
     let cmd = args.get(0).expect("`args` must not be empty");
     let args = &args[1..];
     let path = find_external_subcommand(cmd);
     let command = match path {
         Some(command) => command,
         None => bail!("no such subcommand: `{}`", cmd),
     };
+
+    if let Ok(forc_path) = std::env::current_exe() {
```

Comment:
good idea! If understood correctly we are expecting the forc and the plugin to be in the same directory for them to not emit any warnings. At first I thought this might create a problem with plugins that are hosted in different repos. But if users install them with a single installation source they will end up in the same directory. So this will work nicely and hopefully help users find out that they have shadowed plugins etc happening.